GUI/GUIMod.cs

```diff
@@ -131,7 +131,7 @@ public GUIMod(CkanModule mod, IRegistryQuerier registry, KspVersion current_ksp_
 
             Abstract = mod.@abstract;
             
-            // If we have homepage provided use that, otherwise use the spacedock page or the github repo so that users have somewhere to get more info than just the abstract.
+            // If we have a homepage provided, use that; otherwise use the spacedock page, curse page or the github repo so that users have somewhere to get more info than just the abstract.
 
             Homepage = "N/A";
             if (mod.resources != null)
```

Comment:
Now that we have dropped Mono 3.2 compiler tests we can make this one long `x ?? y ?? z` expression